src/gui/entry/attachments/TextAttachmentsWidget.cpp

```diff
@@ -65,7 +65,15 @@ void TextAttachmentsWidget::updateWidget()
     }
 
     m_editWidget->openAttachment(m_attachment, m_mode);
-    m_previewWidget->openAttachment(m_attachment, attachments::OpenMode::ReadOnly);
```

Comment:
Removing this line causes a delay (almost 1 second) when a preview is rendered, even with really small plain text files. Is this necessary? Could we use something like this here, or does it break the scrollbars you mentioned in the issue thread?
```
if (m_previewWidget->width() > 0) {
    m_previewWidget->openAttachment(m_attachment, attachments::OpenMode::ReadOnly);
}
```

Reply:
This line was removed because it caused the preview widget to be loaded 3 times when the edit window is first shown and the preview isn't even visible. How can removing it cause _more delay_? Not sure on your report here.

Reply:
Ahhhhhh, just preview, not edit. The delay is from the 300ms timer for when the attachment text is changed.